src/main/scala/mesosphere/marathon/core/launcher/OfferProcessor.scala

```diff
@@ -0,0 +1,9 @@
+package mesosphere.marathon.core.launcher
+
+import org.apache.mesos.Protos.Offer
+
+import scala.concurrent.Future
+
+trait OfferProcessor {
```

Comment:
Please add some documentation here.
Especially: who is responsible for the offer matching timeout (caller/callee).